streaming/jms-connector/src/main/java/ActiveMQBroker.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import com.hazelcast.jet.impl.util.ExceptionUtil;
+import org.apache.activemq.broker.BrokerFactory;
+import org.apache.activemq.broker.BrokerService;
+
+import java.net.URI;
+
+/**
+ * Utility class to start/stop an ActiveMQ Broker instance
+ */
+public final class ActiveMQBroker {
+
+    public static final String BROKER_URL = "tcp://localhost:61616";
+
+    private static BrokerService broker;
+
+    public static void start() {
```

Comment:
I know this is just sample code but we should not have obviously not thread safe static methods in our code. Methods and the field can be made non static and broker can be started in the constructor instead.